ui/admin/app/components/auth-methods/auth-method/accounts/account/actions/index.hbs

```diff
@@ -4,7 +4,7 @@
 }}
 
 {{#if (can 'delete account' @model)}}
-  <Hds::Dropdown data-test-manage-account-auth-methods as |dd|>
+  <Hds::Dropdown data-test-manage-account-auth-method as |dd|>
```

Comment:
This `data-test` attribute didn't exactly match so I removed the `s` since this is the account for a single auth-method.